entity-management-tests/src/integration-test/java/eu/europeana/entitymanagement/web/EntityRegistrationIT.java

```diff
@@ -319,16 +319,17 @@ public void registerZohoOrganizationBergerShouldBeSuccessful() throws Exception
   }
 
   @Test
-  void registerOrganizationWithoutWikidataSameAsShouldBeSuccessful() throws Exception {
+  void registerOrganizationWithoutWikidataSameAsAndWithEdmLanguageShouldBeSuccessful()
```

Comment:
please revert back this integration test and create a new one for byzart Organization

Reply:
Done.